src/confd/src/ietf-keystore.c

```diff
@@ -0,0 +1,135 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <sys/stat.h>
+#include <srx/common.h>
+#include <srx/lyx.h>
+
+#include <srx/srx_val.h>
+
+#include "base64.h"
+#include "core.h"
+
+#define XPATH_KEYSTORE_  "/ietf-keystore:keystore/asymmetric-keys"
+
+static size_t filesz(const char *fn)
+{
+	struct stat st;
+
+	if (stat(fn, &st))
+		st.st_size = BUFSIZ;
+	return st.st_size;
+}
+
+/* sanity check, must exist and be of non-zero size */
+static bool fileok(const char *fn)
+{
+	if (!fexist(fn))
+		return false;
+
+	if (filesz(fn) < 42)
```

Comment:
Maybe a comment about why this length was chosen?

Reply:
Fixed, thanks!